### ISSUE TITLE
DEBUG builds crash when leaving Database Icons settings

### ISSUE THREAD

**Issue by vit9696** [keepassium/KeePassium#179]
**Description**
DEBUG builds crash when leaving Database Icons settings

**How to reproduce**
1. Go to Settings → Appearance → Database Icons
2. Return to Appearance screen
3. `KeePassium/NavigationRouter.swift:232: Fatal error
2021-08-22 01:40:48.825973+0300 KeePassium[15827:1873434] KeePassium/NavigationRouter.swift:232: Fatal error`

**Expected behavior**
Appearance screen should show

**Screenshots**

**Environment:**
 - Device: Mac
 - OS: macOS 11.5.1
 - App Version: master

**Comment by keepassium** [comment]
Yep, it's a bug in `NavigationRouter`, I'll get it fixed. Thanks!
